### ISSUE TITLE
Exporting/importing identical copies without From line or Mozilla headers not possible

### ISSUE THREAD

**Issue by pomeloy** [thundernest/import-export-tools-ng#419]
Exporting folders or individual messages results in the From header preceding each message. While this is mandatory to the mbox format, ImportExportTools NG does not seem to adhere to the format outlined in [RFC 4155 (Appendix A)](https://datatracker.ietf.org/doc/html/rfc4155#appendix-A) concerning the From line. 

On re-import, this line is then being added to the imported message or rather not deleted from it. As the From line is peculiar to the mbox format and not a part of the original message, it should not be added to the re-imported email.

Further, the two headers X-Mozilla-Status and X-Mozilla-Status2 are added to the file while not being present in the source file on the IMAP server.

Is there a way to adjust this behavior? This prevents users from backing up and/or restoring an exact copy of a mailbox.

**Comment by cleidigh** [comment]
@pomeloy 
Thanks for the post.
Right now I am buried in the big update for 115. As part of that I am trying to catch new or outstanding issues where possible.
Thunderbird has long standing issues with From escaping and there has been a recent effort to address this.

You are certainly correct that the From separator SHOULD NOT appear in an eml export. I have also observed this as I am doing many tests both with 102 & 115 beta.
In my case this behavior is observed on few out of many exports. I am not minimizing the issue, rather pointing out that as an inconsistent problem it's much more difficult to address.
What is interesting is that the From separator can be observed by viewing source without even exporting. I don't know if that's true 100% of the cases, but it does seem to indicate a potential db issue.

That being said I have exported and reimported such messages apparently without issue. I think Thunderbird has some resiliency in this regard. It sounds like you have messages that might not be handled correctly.

I suspect that IETNG may lack some of the "error handling" work arounds to deal with the back end quirks.

I certainly will try to analyze this more, but some more info from you may help.
Did you look at the message source for any of the problem messages? Have you looked at the consistency % ?

@cleidigh

**Comment by pomeloy** [comment]
Hi and thank you for replying. 

I don't know if that was just a typo on your part, but I am specifically referring to mbox files, not eml ones.

The From header does not appear when viewing the message source of an IMAP inbox, it only appears in the exported files (which is the expected outcome) and subsequently on the reimported messages (which was unexpected). The consistency % is 100.

Obviously this does not impact the basic functionality of just having a manual backup or copy of your emails, it's just that IMHO the reimport thereof should result in an identical copy which is not the case. I first encountered this problem while looking for an automatic IMAP backup solution. Pretty much all of the tools I found are using mbox. So, adding a bogus From header to create a parseable mbox file but not having a way to remove it when reimporting is what I did not expect from an import tool.
